### PR TITLE
Add some tests for event size limits

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: tests/80torture/03events.pl [hunk 1]
```diff
@@ -54,3 +54,32 @@
          params => { from => $user->eventstream_token, timeout => "hello" },
       )->main::expect_http_4xx;
    };
+
+test "Event size limits",
+   requires => do {
+      my $user = local_user_preparer();
+      [ $user, room_preparer( requires_users => [ $user ] ) ];
```
Comment (leonerd) [inline tests/80torture/03events.pl]: FYI, there's `local_user_and_room_preparers` in `10apidoc/33room-members.pl` which does exactly this. You could use that instead.

**Comment by leonerd** [comment]
LGTM, though could use existing function.

File: tests/80torture/03events.pl (hunk 1 above)
Comment (illicitonion) [inline tests/80torture/03events.pl]: Done

**Comment by illicitonion** [comment]
Squash-merged as 526328b

**Pull request closed without merging**